code/modules/lighting/lighting_system.dm

```diff
@@ -58,6 +58,8 @@
 		remove_effect()
 		owner.light = null
 		owner = null
+	if(changed)
+		SSlighting.changed_lights -= src
```

Comment:
This would happen quite frequently and isn't really an error though. All we're doing here is simply cleaning up another reference, which is what Destroy() is supposed to do. The REAL error is when a light like this gets deleted before the round starts and the garbage ss del()'s the light while it's still in the changed_lights list.

This has to happen before the round starts become normally the lighting ss clears the changed_lights list every time it runs. So normally there isn't enough time for the garbage ss to del() before that happens.